README.md

```diff
@@ -35,6 +35,23 @@ You will also need to hash your users' passwords using the new hashing algorithm
 
 ## Client-side usage
 
+### Config
+
+```
+var mediator = require('fh-wfm-mediator');
+
+var config = {
+    forceSessionVerificationOnResume: true
+};
+
+userCore(mediator, config);
+
+```
+
+#### forceSessionVerificationOnResume
```

Comment:
:+1: on no flag, sounds like one of those situations where you need to have stricter security by default